r2-core/src/main/java/com/linkedin/r2/transport/http/client/ratelimiter/Rate.java

```diff
@@ -62,10 +62,18 @@ public Rate(double events, double period, int burst)
       _period = newPeriod;
 
     }
-    else
-    {
-      _events = events;
-      _period = period;
+    else {
```

Comment:
Can we add unit tests to make sure that it works? Also, can we run the simulation in `TestConstantQpsDarkClusterStrategy`?

Reply:
Done